### PR TITLE
Fix for parse_iso_time_tz error with musl library

### PR DESCRIPTION
**Pull request by fcolista**
This patch fixes the error when parsing the time in a format like:
2019-01-18T17:29:00+00:00  with musl.

This behavior is described in the issue #1609

**What**:

Fixes for https://github.com/greenbone/gvmd/issues/1609

### CODE REVIEW

**Comment by timopollmeier** [comment]
Hello, @fcolista! Thank you for your patch. It looks good to me, but could you also add a changelog entry in the 20.08 section, please?

**Comment by fcolista** [comment]
hi @timopollmeier thanks for your feedback!
Just to be sure, this is what you mean?
```
x86_64-edge:~/gitrepos/github/gvmd$ git diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index d3557daba..3ee1d4722 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Improve VT version handling for CVE & OVAL results [#1496](https://github.com/greenbone/gvmd/pull/1496)
 - Fix migration to DB version 242 from gvmd 20.08 [#1498](https://github.com/greenbone/gvmd/pull/1498)
 - Update subject alternative name in certificate generation [#1503](https://github.com/greenbone/gvmd/pull/1503)
+- Fix for parse_iso_time_tz error with musl library [#1644] (https://github.com/greenbone/gvmd/pull/1644)
 
 ### Removed
 - Remove Network Source Interface from gvmd [#1511](https://github.com/greenbone/gvmd/pull/1511)
 ```

**Comment by Dexus** [comment]
> hi @timopollmeier thanks for your feedback!
> Just to be sure, is this what you mean?
> 
> ```
> x86_64-edge:~/gitrepos/github/gvmd$ git diff
> diff --git a/CHANGELOG.md b/CHANGELOG.md
> index d3557daba..3ee1d4722 100644
> --- a/CHANGELOG.md
> +++ b/CHANGELOG.md
> @@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
>  - Improve VT version handling for CVE & OVAL results [#1496](https://github.com/greenbone/gvmd/pull/1496)
>  - Fix migration to DB version 242 from gvmd 20.08 [#1498](https://github.com/greenbone/gvmd/pull/1498)
>  - Update subject alternative name in certificate generation [#1503](https://github.com/greenbone/gvmd/pull/1503)
> +- Fix for parse_iso_time_tz error with musl library [#1644](https://github.com/greenbone/gvmd/pull/1644)
>  
>  ### Removed
>  - Remove Network Source Interface from gvmd [#1511](https://github.com/greenbone/gvmd/pull/1511)
> ```

Hi @fcolista he means you should add it to https://github.com/greenbone/gvmd/blob/master/CHANGELOG.md#2083-unreleased as it will backported to v20.8.3 and v21.4.3

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Fix sending prefs for whole, growing VT families [#1603](https://github.com/greenbone/gvmd/pull/1603)
 - Add trash columns for target "elevate" credential [#1636](https://github.com/greenbone/gvmd/pull/1636)
+- Fix for parse_iso_time_tz error with musl library [#1644](https://github.com/greenbone/gvmd/pull/1644)
```
Comment (bjoernricks) [inline CHANGELOG.md:39]:
```suggestion
```

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**